src/Illuminate/Database/Grammar.php

```diff
@@ -81,6 +82,10 @@ public function wrap($value)
             return $this->getValue($value);
         }
 
+        if ($value instanceof BackedEnum) {
+            $value = $value->value;
+        }
```

Comment:
You could have used [`Illuminate\Support\enum_value()`](https://github.com/laravel/framework/blob/11.x/src/Illuminate/Collections/functions.php):
```suggestion
        $value = enum_value($value);
```